test/createInput.test.ts

```diff
@@ -45,7 +38,7 @@ describe("createInput", () => {
   })
 
   test("it replays the latest value to new subscriptions", () => {
```

Comment:
Food for thought: should it?

Reply:
Good point! Now that it's stateless, it shouldn't, no!